doc/modules/model_evaluation.rst

```diff
@@ -1277,6 +1278,13 @@ function:
     >>> matthews_corrcoef(y_true, y_pred)
     -0.33...
 
+.. topic:: References:
+
+  .. [WikipediaMCC2021] Wikipedia contributors. Detection error tradeoff.
```

Comment:
Uhm there is something wrong here since it specify the DET instead of the MCC?

Reply:
Right. Changed that, and using the Phi coefficient page directly since MCC is redirected here.